zookeeper-server/src/main/java/org/apache/zookeeper/server/backup/storage/impl/FileSystemBackupStorage.java

```diff
@@ -166,11 +168,15 @@ public void copyToBackupStorage(File srcFile, File destName) throws IOException
           Paths.get(BackupStorageUtil.constructBackupFilePath(destName.getName(), fileRootPath)),
           StandardCopyOption.REPLACE_EXISTING);
     } finally {
-      if (inputStream != null) {
-        inputStream.close();
-      }
-      if (outputStream != null) {
-        outputStream.close();
+      try {
```

Comment:
Could you please help me understand what error is it in this place? Why do we want try-catch inside finally block?